patroni/dcs/__init__.py

```diff
@@ -458,9 +490,20 @@ def cancel_initialization(self):
         """ Removes the initialize key for a cluster """
 
     @abc.abstractmethod
-    def delete_cluster(self):
+    def delete_cluster(self, index=None):
```

Comment:
Basically yes. Looking at the diff there isn't much that would make sense as a separate patch besides protecting touch_member in a lock.